packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -198,7 +199,10 @@ class AotToolsArtifact extends CachedArtifact {
   String get name => 'aot-tools';
```

Comment:
Should we rename this to executableName to avoid confusion? I don't think it's obvious what the difference between name and fileName is at first glance.

Reply:
I can add some comments. I changed `executables` to `executable` (because our artifacts only ever have one file) to `fileName` (because our artifacts aren't always executable).